app/views/rejected_notification_errors.scala.html

```diff
@@ -35,21 +35,25 @@
     <span id="information">@messages("rejected.notification.information")</span>
 
     <table>
-        <tr class="table-row">
-            <th class="table-cell table-head">@messages("rejected.notification.header.fieldName")</th>
-            <th class="table-cell table-head">@messages("rejected.notification.header.errorCode")</th>
-            <th class="table-cell table-head">@messages("rejected.notification.header.errorDescription")</th>
-        </tr>
-
-        @for((error, index) <- errors.zipWithIndex) {
-          <tr class="table-row" id="rejected_notifications-row-@index">
-            <th class="table-cell table-head" id="rejected_notifications-row-@index-name">
-                @error.pointer.map(messages(_))
-            </th>
-            <th class="table-cell" id="rejected_notifications-row-@index-code">@error.code</th>
-            <th class="table-cell" id="rejected_notifications-row-@index-description">@error.description</th>
-          </tr>
-        }
+        <thead>
+            <tr class="table-row">
+                <th class="table-cell table-head">@messages("rejected.notification.header.fieldName")</th>
```

Comment:
please add class for columns names - like "field name" would be much easier to manage test

Reply:
I'll add class name, but I can't agree that it'll be easier, those elements has unique id and `field-name` can be used somewhere in the css. I'll add `rejected-field-name` to be sure that this change doesn't change the view or doesn't apply styles that we don't want.

Reply:
Changed